resources/github-actions.md

```diff
@@ -176,51 +176,144 @@ The example code below demonstrates building and testing with conda packages fro
 Replace the pull request numbers and library names as needed.
 Remember that changes to use CI artifacts should be _temporary_ and should be reverted prior to merging any required changes in that PR.
 
-**Example 1:** Building `libcuml` (C++) using `librmm`, `libraft`, `libcumlprims_mg` PR artifacts.
+**Example 1:** Building `libcuml` (C++) using `librmm` and `libraft` PR artifacts.
 
-```sh
-# ci/build_cpp.sh
+Add a new file called `ci/use_conda_packages_from_prs.sh`.
 
-LIBRMM_CHANNEL=$(rapids-get-pr-conda-artifact rmm 1095 cpp)
+```shell
+# ci/use_conda_packages_from_prs.sh
+
+# download CI artifacts
 LIBRAFT_CHANNEL=$(rapids-get-pr-conda-artifact raft 1388 cpp)
-LIBCUMLPRIMS_CHANNEL=$(rapids-get-pr-conda-artifact cumlprims_mg 129 cpp)
+LIBRMM_CHANNEL=$(rapids-get-pr-conda-artifact rmm 1095 cpp)
+
+# make sure they can be found locally
+conda config --system --add channels "${LIBRAFT_CHANNEL}"
+conda config --system --add channels "${LIBRMM_CHANNEL}"
```

Comment:
Note: this will change with rattler-build. Should we preface this with that disclaimer?

Reply:
I'd written this before I knew that: https://github.com/rapidsai/gha-tools/pull/146#issuecomment-2674948061

I don't think we should add a disclaimer like that. A note of the form "this won't work for builds using `rattler-build`" isn't something that folks reading this could do much with, by themselves.

It'd be better to add "update the docs on using CI artifacts" as part of the work for https://github.com/rapidsai/build-planning/issues/47